### DISCUSSION TITLE
Xpress management page

### DISCUSSION THREAD

**Discussion by simonfernandes** [UserOfficeProject/user-office-project-issue-tracker#1033] [General]
Xpress scientists need to be able to manage Xpress proposals. 
This discussion should be used to track the design decisions we make for the Xpress management page, the final decisions should be tracked in this top comment.   

**Questions:**

Where should we put the management page? 

- The management page will be on the New Proposal software

How will we include sample can management? 

How will we manage the comments Xpress scientists can leave on proposals?

How will we make sure a scientist can only see/ assign an instrument to/ change the status of proposals for their techniques? 
 
**Features needed (not exhaustive):**

- Adding and saving comments against a proposal, visible to scientists only (https://github.com/UserOfficeProject/issue-tracker/issues/1031)
- Assigning an instrument based from limited list of instruments only includes those that are both assigned to the call and technique
- Changing the status of proposals from a static list
- Status changes should trigger relevant notifications and ingestion
- Export proosals to a file (https://github.com/UserOfficeProject/issue-tracker/issues/1028)

**Comment by ellen-wright** [comment]
If we can use existing functionality in the new software with the proposal table and managing status of proposals then my vote would be to keep everything in one place in the new software
